sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/ConditionEvaluatesToConstant.cs

```diff
@@ -1803,6 +1803,30 @@ public void NonCompliantMethod3(bool? input, bool input1)
         Console.WriteLine("input is true");
       }
     }
+  }
+
+  class Program
+  { 
+    public static string CompliantMethod4(string parameter)
+    {
+      var useParameter = parameter ?? "non-empty";
+      if (useParameter == null || useParameter=="")  // Noncompliant FP #1347
+        return "non-empty"; // Secondary
```

Comment:
we should improve the error message for the missing secondary locations... this sprint...

Reply:
I totally agree